rules/gcp_audit_rules/gcp_dns_zone_modified_or_deleted.py

```diff
@@ -1,5 +1,5 @@
-from panther_base_helpers import deep_get
 from gcp_base_helpers import gcp_alert_context
+from panther_base_helpers import deep_get
```

Comment:
I assume this is from running the formatter

Reply:
yep, it was. I think I pulled this in my recent merge as well.